spec/index.html

```diff
@@ -297,15 +297,48 @@ <h3>IRIs</h3>
     <h3>Literals</h3>
 
     <p><a data-cite="RDF12-CONCEPTS#dfn-literal">Literals</a>
-    are basic values that are not IRIs. Examples of literals include
-    strings such as "La Joconde", dates such as "the 4th of July, 1990"
-    and numbers such as "3.14159".
-    Literals are associated with a <a data-cite="RDF12-CONCEPTS#dfn-datatype">datatype</a>
-    enabling such values to be parsed and interpreted correctly.
-    String literals can optionally be associated with a <a data-cite="RDF12-CONCEPTS#dfn-language-tag">language tag</a>.
-    For example, "Léonard de Vinci" could
-    be associated with the "fr" language tag and "李奥纳多·达·文西"
-    with the "zh" language tag.</p>
+    are used to encode values, rather than identifying them by IRIs. Examples
+    of literals include strings such as "La Joconde", dates such as
+    "1990-07-04" (the 4th of July, 1990) and numbers such as "3.14159".
+    Literals are always associated with a
+    <a data-cite="RDF12-CONCEPTS#dfn-datatype">datatype</a>, enabling them to
+    be parsed and interpreted correctly.</p>
+
+    <p>String literals can optionally be associated with a <a
```

Comment:
I don't think these are called "string literals" in RDF Concepts. A "Simple Literal" has datatype xsd:string, and as you note with language tag possibly with base direction, "language-tagged string" or "directional language-tagged string".

Reply:
I agree and I'd prefer a concept for the two kinds of strings paired with a language (e.g. "text literals"). But this usage was [already in the 1.1 Primer](https://www.w3.org/TR/rdf11-primer/#section-literal), with:
> String literals can optionally be associated with a language tag 

Perhaps just saying "Strings" instead would be good enough?